cmake/README.md

```diff
@@ -43,7 +43,35 @@ The following targets are available for build system after configuration:
 Sudirectories are added as labels for each test and can be used with `ctest -L <label>`.
 For example, `<root>/test/path/to/test.pass.cpp` will have `path` and `to` labels.
 
-## How to use oneDPL package from CMake
+## How to use oneDPL from CMake
+### Using oneDPL source files
+
+This way allow to integrate oneDPL source code into user project using the [add_subdirectory](https://cmake.org/cmake/help/latest/command/add_subdirectory.html) command. `add_subdirectory(<oneDPL_root_dir> <oneDPL_output_dir>)`, where `<oneDPL_root_dir>` is a relative or absolute path to oneDPL root dir and `<oneDPL_output_dir>` is a relative or absolute path to directory for holding output files of oneDPL, adds oneDPL to user project build. If `<oneDPL_root_dir>` is the relative path, then `<oneDPL_output_dir>` is the optional variable.
+
+For example:
+
+```cmake
+project(Foo)
+add_executable(foo foo.cpp)
+
+# Add oneDPL to the build.
+add_subdirectory(/path/to/oneDPL /path/to/build_oneDPL)
+```
+When using this way oneDPL is built with the user project simultaneously, so variables affecting oneDPL build can be specified in the user project's CMakeLists.txt file:
+
+```cmake
+project(Foo)
+add_executable(foo foo.cpp)
+
+# Add oneDPL to the build.
+add_subdirectory(/path/to/oneDPL /path/to/build_oneDPL)
+
+# Specify oneDPL backend
+target_compile_definitions(foo PRIVATE ONEDPL_BACKEND=tbb)
```

Comment:
`ONEDPL_BACKEND` is not compile definition. but CMake variable.
It should be initialized using `set` _before_ `add_subdirectory`.

Reply:
Did not notice. Fixed.